project/Common.scala

```diff
@@ -26,8 +26,6 @@ object Common extends AutoPlugin {
 
     licenses := Seq(("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0"))),
 
-    scalaVersion := crossScalaVersions.value.head,
-    crossScalaVersions := Dependencies.ScalaVersions,
```

Comment:
um, suddenly travis becomes part of the build files.
OK, lets do that, but keep a comment here saying that those come from travis via (what?) plugin